src/hooks/use-proposals.js

```diff
@@ -97,8 +98,36 @@ const useProposals = () => {
 
       if (!oldProposal) return _proposal
 
+      const totalVotingPnt = _proposal.yes.plus(_proposal.no).plus(oldProposal.yes).plus(oldProposal.no)
+      const totalYes = _proposal.yes.plus(oldProposal.yes)
+      const totalNo = _proposal.no.plus(oldProposal.no)
+      const percentageYea = totalYes.dividedBy(totalVotingPnt).multipliedBy(100)
+      const percentageNay = totalNo.dividedBy(totalVotingPnt).multipliedBy(100)
+      const totalYesPercentage = formatAssetAmount(percentageYea, '%', { decimals: 2 })
+      const totalNoPercentage = formatAssetAmount(percentageNay, '%', { decimals: 2 })
+
+      const minAcceptQuorum =
+        _proposal.minAcceptQuorum > oldProposal.minAcceptQuorum
+          ? _proposal.minAcceptQuorum
+          : oldProposal.minAcceptQuorum
+
+      const totalVotingPower = _proposal.votingPower.plus(oldProposal.votingPower)
+      const totalQuorum = _proposal.yes.plus(oldProposal.yes).dividedBy(totalVotingPower)
+      const biggestMinQuorum = minAcceptQuorum
+      const totalQuorumReached = totalQuorum.isGreaterThan(biggestMinQuorum)
+      const totalPassed = percentageYea.isGreaterThan(51) && totalQuorumReached
```

Comment:
51? or 50? you are using greater than

Reply:
This is the same computation done up tonow for v2 only proposals. I am currently not able to tell if this is a feature or a bug 😅

Reply:
51 should be correct: https://polygonscan.com/address/0xb1b013d9ffba9b19be1b4bf7a489f0cc5d96f6ec#readProxyContract#F30

Reply:
Ok, makes sense now. Thanks for checking